src/workspace-data/data-table/uri-viewer/UriPreview.js

```diff
@@ -22,23 +24,34 @@ const styles = {
   },
 };
 
-const isImage = ({ contentType, name }) => {
-  return /^image/.test(contentType) || /\.(?:(jpe?g|png|svg|bmp))$/.test(name);
+export const isImage = ({ contentType, name }) => {
+  return /^(?:image)/.test(contentType) || /\.(?:jpe?g|png|svg|bmp)$/.test(name);
 };
 
-const isText = ({ contentType, name }) => {
-  return /(?:(^text|application\/json))/.test(contentType) || /\.(?:(txt|[ct]sv|log|json))$/.test(name);
+export const isText = ({ contentType, name }) => {
+  return /^(?:text|application\/json)/.test(contentType) || /\.(?:txt|[ct]sv|log|json|fastq|fasta|fa|vcf|sam|bed|interval_list|gtf|md)$/.test(name);
 };
 
-const isBinary = ({ contentType, name }) => {
-  // excluding json and google's default types (i.e. wasn't set to anything else)
+export const isBinary = ({ contentType, name }) => {
   return (
-    /application(?!\/(json|octet-stream|x-www-form-urlencoded)$)/.test(contentType) || /(?:(\.(?:(ba[mi]|cra[mi]|pac|sa|bwt|gz))$|\.gz\.))/.test(name)
+    /application(?!\/(?:json|octet-stream|x-www-form-urlencoded)$)/.test(contentType) || /(?:\.(?:ba[mi]|cra[mi]|pac|sa|bwt|bcf|h5ad)$)/.test(name)
   );
 };
```

Comment:
previously, `isBinary` was used as part of `isFilePreviewable`. In this PR, `isBinary` is not used at all (except in tests). If we truly don't need it, could you remove it and its tests? I don't know if it is necessary to put it back in to `isFilePreviewable`.

Reply:
Thanks, I have [removed isBinary and its tests](https://github.com/DataBiosphere/terra-ui/pull/5029/commits/8918d96b09c0c37a902faabc53dae6811632369f).